advisor/pkg/api/pod_traffic.go

```diff
@@ -66,13 +66,13 @@ func GetPodTraffic(podName string) ([]PodTraffic, error) {
 
 	// Parse the JSON response and unmarshal it into the Go struct.
 	if err := json.Unmarshal([]byte(body), &podTraffic); err != nil {
-		log.Error().Err(err).Msg("Error unmarshal JSON")
+		log.Warn().Err(err).Msg("Error unmarshal JSON")
```

Comment:
I think I should return this to being an error or potentially even fatal as this will break